src/core/component.ts

```diff
@@ -232,6 +232,22 @@ const BaseComponent: VueConstructor<IBaseComponent> = Vue.extend({
     }
 });
 
+function getElement(createElement, children, id) {
```

Comment:
Not sure this should be extracted as a `function` - used once, one `return` statement, three lines